syz-manager/manager.go

```diff
@@ -109,6 +109,7 @@ type Crash struct {
 	vmIndex int
 	hub     bool // this crash was created based on a repro from hub
 	*report.Report
+	machineInfo []byte
```

Comment:
If we remove the field here then we will probably need to extract the machine info inside `manager.saveCrash`, and we will need to delay re-spawning the vm after we get the info to avoid a race condition. From what I know, copying go slice only does a shallow copy and is not an expensive operation, so I think the copy here is not so bad?